package.json

```diff
@@ -10,30 +10,52 @@
   "license": "MIT",
   "author": "Sanity.io <hello@sanity.io>",
   "sideEffects": false,
+  "type": "commonjs",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/1.esm.d.ts",
-        "default": "./dist/1.esm.mjs"
+      "types": "./dist/index.d.ts",
+      "source": "./src/1.ts",
+      "require": "./dist/index.js",
+      "node": {
+        "import": "./dist/index.cjs.mjs",
+        "require": "./dist/index.js"
       },
-      "require": {
-        "types": "./dist/1.umd.d.ts",
-        "default": "./dist/1.umd.cjs"
+      "import": "./dist/index.mjs",
+      "default": "./dist/index.js"
+    },
+    "./1": {
+      "types": "./dist/1.d.ts",
+      "source": "./src/1.ts",
+      "require": "./dist/1.js",
+      "node": {
+        "import": "./dist/1.cjs.mjs",
+        "require": "./dist/1.js"
       },
-      "default": "./dist/1.umd.cjs"
+      "import": "./dist/1.mjs",
+      "default": "./dist/1.js"
     },
     "./package.json": "./package.json"
   },
-  "main": "./dist/1.umd.cjs",
-  "types": "./dist/1.umd.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "source": "./src/1.ts",
+  "types": "./dist/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "1": [
```

Comment:
This is necessary for TypeScript to find the type definitions for `import {parse, evaluate} from 'groq/1'` imports. TS doesn't yet support `pkg.exports.types`. TypeScript's new `Node16` mode supports `pkg.exports.types` but it doesn't count as the way it requires you to write ESM import paths isn't compatible with other tooling like ESLint, babel, rollup and other bundlers.